inference-engine/src/vpu/graph_transformer/src/stages/static_shape_nms.cpp

```diff
@@ -74,11 +76,35 @@ class StaticShapeNMS final : public StageNode {
         input5->serializeBuffer(serializer);
         outputData->serializeBuffer(serializer);
         outputDims->serializeBuffer(serializer);
+
+        if (tempBuffers().size())
+            tempBuffer(0)->serializeBuffer(serializer);
     }
 };
 
+bool isCMXEnough(int cmxSize, int numSlices, std::vector<int> bufferSizes) {
```

Comment:
Probably I missed something, but could this function be replaced with check

```
const auto total = std::accumulate(bufferSizes.cbegin(), bufferSizes.cend(), 0);
return total < cmxSize * numSlices;
```

Reply:
No it cannot, because memory slices that SHAVEs can use are not continuous. That is the reason that this function simulates memory allocation to check if data can be kept in CMX.

Reply:
Thanks for explanation